magicctapipe/scripts/lst1_magic/semi_automatic_scripts/coincident_events.py

```diff
@@ -92,6 +93,8 @@ def linking_bash_lst(target_dir, LST_runs, source_name, LST_version, env_name, c
         Name of the conda environment
     cluster : str
         Cluster system
+    Nice_parameter : int
```

Comment:
it should be consistent with slurm_lines, so if int or None is allowed there it should be also in this function.
also in python convesions normally the variables start with lower letter, so ```nice_parameter``` like in ```slurm_lines```, or even better simply ```nice``